samples/c2pa/plugin-dash/c2pa-plugin.js

```diff
@@ -24,7 +24,7 @@ async function c2pa_init(player, onPlaybackTimeUpdated) {
                 let tag = chunk.streamId + '-' + chunk.mediaInfo.type + '-' + chunk.representationId;
 
                 if (chunk.segmentType == 'InitializationSegment') {
-                    initFragment[tag] = new Blob([chunk.bytes], {type: chunk.mediaInfo.mimeType});
+                    initFragment[tag] = new Blob([chunk.bytes], {type: 'video/mp4'});
```

Comment:
Is it always gonna be `video/mp4`?

Reply:
Yeah I wanted to bring this up - I think in the native implementation, we set this if to video/mp4. I thought chunk.mediaInfo.mimeType was a better way to handle this, but when testing the tears of steel video, I get an error on the audio chunks. I need to investigate a bit better

Reply:
Added todo in code